tests/api/bootstrap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+error_reporting(E_ALL);
+
+if (getenv('DD_AUTOLOAD_NO_COMPILE') == 'true' && (false !== getenv('CI') || false !== getenv('CIRCLECI'))) {
+    throw new Exception('Running uncopiled files in CI');
```

Comment:
Nit: s/uncopiled/uncompiled/ or perhaps: `Tests must run using the _generated.php script in CI`